meteor-app/imports/api/orders/schema.test.js

```diff
@@ -0,0 +1,92 @@
+// schema.test.js
+
+/* eslint-disable no-unused-expressions */
+
+/**
+ * schema test
+ * we probably don't need to unit test all the schemas but I put this
+ * suite together so i could understand how SimpleSchema.autoValue() actually
+ * works and where the holes are.
+ */
+
+import { resetDatabase } from '/imports/test/util-test'
+import { expect } from 'chai'
+import { Random } from 'meteor/random'
+
+import Orders from '/imports/api/orders/schema'
+import Factory from '/imports/test/factories'
+import { RegExId } from '../schema';
+
+const badOrders = [
```

Comment:
Please add more test cases that should fail. You should have some test cases that exercise the parts sub-schema as well.